packages/common/src/condaStore.ts

```diff
@@ -1,8 +1,11 @@
+import {Conda} from '../src/tokens'
+
+
 export namespace CondaStore {
     /**
      * Description of the REST API response for each environment
      */
-    export interface IEnvironment {
+    export interface IEnvironment extends Conda.IEnvironment {
```

Comment:
Note to self: Do `dir` and `is_default` in `Conda.IEnvironment` also make sense in conda-store context? need to check difference between conda and conda-store envirnments

Reply:
Yeah, they might make sense here. We can always change this later, thanks to making it a subclass!